### PR TITLE
added importing multiple url

### PR DESCRIPTION
**Pull request by liutiming**
as discussed

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -243,51 +257,58 @@ def getCookies(self):
...
-        if urllib.parse.urlparse(url).scheme:
-            urlDomain = urllib.parse.urlparse(url).netloc
-            urlPath = urllib.parse.urlparse(url).path
-        else:
-            urlDomain = urllib.parse.urlparse("https://"+url).netloc
-            urlPath = urllib.parse.urlparse("https://"+url).path
+        report = {'error': [], 'success': []}
+        urls = self.text_url.text().splitlines()
+        self.label_results.setText(("There are <b>{0}</b> urls in total. Starting".format(len(urls))))
+        self.sleep(0.5)
```
Comment (kelciour) [inline __init__.py:265]: It looks like `self.sleep(0.5)` was added by mistake and can be removed.

**Review by kelciour — CHANGES_REQUESTED** [review]
Could you please also move down Parent deck name? Maybe something like this.

From

image

to 

image

I think it's a bit confusing to ask for the parent deck name at first.

**Comment by kelciour** [comment]
Thank you for the pull request! I was able to import multiple Quizlet urls separated by a new line by copypasting them from the text file.

**Comment by liutiming** [comment]
Nice and thanks! updated changes. The UI isn't prettiest. Is there a way to auto-wrap the instructions?

**Comment by kelciour** [comment]
Thank you for the changes! Maybe https://stackoverflow.com/a/12281629 but I haven't tried it and it already looks good to me.

**Pull request merged**